src/js/store/orgs/actions.ts

```diff
@@ -348,3 +353,37 @@ export const commitFailed = ({
     payload: model,
   });
 };
+
+export const refreshOrg = (org: Org): ThunkResult => async (dispatch) => {
+  const url = window.api_urls.scratch_org_refresh(org.id);
+  dispatch({
+    type: 'REFRESH_ORG_STARTED',
+    payload: { org, url },
+  });
+  try {
+    const response = await apiFetch({
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      url: addUrlParams(url),
```

Comment:
No need to call `addUrlParams` when we're not actually adding any query string parameters.